account_invoice_transmit_method/views/partner.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+  Copyright 2017-2020 Akretion France (http://www.akretion.com/)
+  @author: Alexis de Lattre <alexis.delattre@akretion.com>
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+<odoo>
+    <record id="view_partner_property_form" model="ir.ui.view">
+        <field
+            name="name"
+        >Add Invoice Transmit Methods on partner form view (account tab)</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="account.view_partner_property_form" />
+        <field name="arch" type="xml">
+            <field name="property_payment_term_id" position="after">
+                <field name="customer_invoice_transmit_method_id" widget="selection" />
```

Comment:
any good reason to use `selection`? If you have several records here this is going to be not so user friendly.

Reply:
Well, it would be surprising to have many transmit methods ; we usually only have a few. That's why I put widget=selection.